pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.ecocode</groupId>
     <artifactId>ios-plugin</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.1.0</version>
```

Comment:
for me, this is the releasing process that will do this king of modifications.
for me, keep "SNAPSHOT" version here and check if releasing process is OK with ios release (or upgrade it)

Reply:
By using the "[Maven Friendly Version](https://maven.apache.org/maven-ci-friendly.html)" guide, there is no more file modification needed in Git for a release.

Reply:
@jycr I didn't know this maven feature ! cool feature indeed ! ... but we have to refactor our pom.xml files and github actions for release system ... maybe an improvment issue ? :p

Reply:
> By using the "[Maven Friendly Version](https://maven.apache.org/maven-ci-friendly.html)" guide, there is no more file modification needed in Git for a release.

I created an issue in green-code-initiative/ecoCode#208
We might migrate this feature to other repositories if it proves interesting